[2024-12-06T18:16:07.772+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-06T18:16:07.796+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: pyspark_etl_dag.run_pyspark_etl manual__2024-12-06T18:06:28.556095+00:00 [queued]>
[2024-12-06T18:16:07.802+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: pyspark_etl_dag.run_pyspark_etl manual__2024-12-06T18:06:28.556095+00:00 [queued]>
[2024-12-06T18:16:07.803+0000] {taskinstance.py:2866} INFO - Starting attempt 2 of 1
[2024-12-06T18:16:07.812+0000] {taskinstance.py:2889} INFO - Executing <Task(SparkSubmitOperator): run_pyspark_etl> on 2024-12-06 18:06:28.556095+00:00
[2024-12-06T18:16:07.816+0000] {standard_task_runner.py:72} INFO - Started process 180 to run task
[2024-12-06T18:16:07.818+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'pyspark_etl_dag', 'run_pyspark_etl', 'manual__2024-12-06T18:06:28.556095+00:00', '--job-id', '46', '--raw', '--subdir', 'DAGS_FOLDER/scripts_etl_dags.py', '--cfg-path', '/tmp/tmpjg_vsc5r']
[2024-12-06T18:16:07.821+0000] {standard_task_runner.py:105} INFO - Job 46: Subtask run_pyspark_etl
[2024-12-06T18:16:07.845+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.9/site-packages/***/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-12-06T18:16:07.890+0000] {task_command.py:467} INFO - Running <TaskInstance: pyspark_etl_dag.run_pyspark_etl manual__2024-12-06T18:06:28.556095+00:00 [running]> on host 468951cd467e
[2024-12-06T18:16:07.988+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='pyspark_etl_dag' AIRFLOW_CTX_TASK_ID='run_pyspark_etl' AIRFLOW_CTX_EXECUTION_DATE='2024-12-06T18:06:28.556095+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-06T18:06:28.556095+00:00'
[2024-12-06T18:16:07.989+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-06T18:16:08.015+0000] {base.py:84} INFO - Retrieving connection 'spark_default'
[2024-12-06T18:16:08.016+0000] {spark_submit.py:473} INFO - Spark-Submit cmd: spark-submit --master yarn --conf spark.executor.memory=2g --conf spark.driver.memory=2g --name arrow-spark --queue root.default /opt/***/scripts/scripts_etl.py
[2024-12-06T18:16:10.303+0000] {spark_submit.py:634} INFO - 24/12/06 18:16:10 INFO SparkContext: Running Spark version 3.4.4
[2024-12-06T18:16:10.355+0000] {spark_submit.py:634} INFO - 24/12/06 18:16:10 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
[2024-12-06T18:16:10.442+0000] {spark_submit.py:634} INFO - 24/12/06 18:16:10 INFO ResourceUtils: ==============================================================
[2024-12-06T18:16:10.443+0000] {spark_submit.py:634} INFO - 24/12/06 18:16:10 INFO ResourceUtils: No custom resources configured for spark.driver.
[2024-12-06T18:16:10.444+0000] {spark_submit.py:634} INFO - 24/12/06 18:16:10 INFO ResourceUtils: ==============================================================
[2024-12-06T18:16:10.445+0000] {spark_submit.py:634} INFO - 24/12/06 18:16:10 INFO SparkContext: Submitted application: PostgreSQL Integration
[2024-12-06T18:16:10.466+0000] {spark_submit.py:634} INFO - 24/12/06 18:16:10 INFO ResourceProfile: Default ResourceProfile created, executor resources: Map(cores -> name: cores, amount: 1, script: , vendor: , memory -> name: memory, amount: 2048, script: , vendor: , offHeap -> name: offHeap, amount: 0, script: , vendor: ), task resources: Map(cpus -> name: cpus, amount: 1.0)
[2024-12-06T18:16:10.480+0000] {spark_submit.py:634} INFO - 24/12/06 18:16:10 INFO ResourceProfile: Limiting resource is cpus at 1 tasks per executor
[2024-12-06T18:16:10.482+0000] {spark_submit.py:634} INFO - 24/12/06 18:16:10 INFO ResourceProfileManager: Added ResourceProfile id: 0
[2024-12-06T18:16:10.534+0000] {spark_submit.py:634} INFO - 24/12/06 18:16:10 INFO SecurityManager: Changing view acls to: ***
[2024-12-06T18:16:10.536+0000] {spark_submit.py:634} INFO - 24/12/06 18:16:10 INFO SecurityManager: Changing modify acls to: ***
[2024-12-06T18:16:10.537+0000] {spark_submit.py:634} INFO - 24/12/06 18:16:10 INFO SecurityManager: Changing view acls groups to:
[2024-12-06T18:16:10.537+0000] {spark_submit.py:634} INFO - 24/12/06 18:16:10 INFO SecurityManager: Changing modify acls groups to:
[2024-12-06T18:16:10.538+0000] {spark_submit.py:634} INFO - 24/12/06 18:16:10 INFO SecurityManager: SecurityManager: authentication disabled; ui acls disabled; users with view permissions: ***; groups with view permissions: EMPTY; users with modify permissions: ***; groups with modify permissions: EMPTY
[2024-12-06T18:16:10.730+0000] {spark_submit.py:634} INFO - 24/12/06 18:16:10 INFO Utils: Successfully started service 'sparkDriver' on port 46635.
[2024-12-06T18:16:10.766+0000] {spark_submit.py:634} INFO - 24/12/06 18:16:10 INFO SparkEnv: Registering MapOutputTracker
[2024-12-06T18:16:10.797+0000] {spark_submit.py:634} INFO - 24/12/06 18:16:10 INFO SparkEnv: Registering BlockManagerMaster
[2024-12-06T18:16:10.814+0000] {spark_submit.py:634} INFO - 24/12/06 18:16:10 INFO BlockManagerMasterEndpoint: Using org.apache.spark.storage.DefaultTopologyMapper for getting topology information
[2024-12-06T18:16:10.815+0000] {spark_submit.py:634} INFO - 24/12/06 18:16:10 INFO BlockManagerMasterEndpoint: BlockManagerMasterEndpoint up
[2024-12-06T18:16:10.847+0000] {spark_submit.py:634} INFO - 24/12/06 18:16:10 INFO SparkEnv: Registering BlockManagerMasterHeartbeat
[2024-12-06T18:16:10.864+0000] {spark_submit.py:634} INFO - 24/12/06 18:16:10 INFO DiskBlockManager: Created local directory at /tmp/blockmgr-b508700c-37c9-4695-ac3a-ba6efc0fa7ab
[2024-12-06T18:16:10.880+0000] {spark_submit.py:634} INFO - 24/12/06 18:16:10 INFO MemoryStore: MemoryStore started with capacity 1048.8 MiB
[2024-12-06T18:16:10.910+0000] {spark_submit.py:634} INFO - 24/12/06 18:16:10 INFO SparkEnv: Registering OutputCommitCoordinator
[2024-12-06T18:16:11.026+0000] {spark_submit.py:634} INFO - 24/12/06 18:16:11 INFO JettyUtils: Start Jetty 0.0.0.0:4040 for SparkUI
[2024-12-06T18:16:11.077+0000] {spark_submit.py:634} INFO - 24/12/06 18:16:11 INFO Utils: Successfully started service 'SparkUI' on port 4040.
[2024-12-06T18:16:11.108+0000] {spark_submit.py:634} INFO - 24/12/06 18:16:11 INFO SparkContext: Added JAR /opt/***/scripts/postgresql-42.7.4.jar at spark://468951cd467e:46635/jars/postgresql-42.7.4.jar with timestamp 1733508970290
[2024-12-06T18:16:11.238+0000] {spark_submit.py:634} INFO - 24/12/06 18:16:11 INFO DefaultNoHARMFailoverProxyProvider: Connecting to ResourceManager at /0.0.0.0:8032
[2024-12-06T18:16:12.364+0000] {spark_submit.py:634} INFO - 24/12/06 18:16:12 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:16:13.365+0000] {spark_submit.py:634} INFO - 24/12/06 18:16:13 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:16:14.366+0000] {spark_submit.py:634} INFO - 24/12/06 18:16:14 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:16:15.367+0000] {spark_submit.py:634} INFO - 24/12/06 18:16:15 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:16:16.368+0000] {spark_submit.py:634} INFO - 24/12/06 18:16:16 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:16:17.369+0000] {spark_submit.py:634} INFO - 24/12/06 18:16:17 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:16:18.371+0000] {spark_submit.py:634} INFO - 24/12/06 18:16:18 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:16:19.372+0000] {spark_submit.py:634} INFO - 24/12/06 18:16:19 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:16:20.374+0000] {spark_submit.py:634} INFO - 24/12/06 18:16:20 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:16:21.376+0000] {spark_submit.py:634} INFO - 24/12/06 18:16:21 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:16:22.384+0000] {spark_submit.py:634} INFO - 24/12/06 18:16:22 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:16:23.386+0000] {spark_submit.py:634} INFO - 24/12/06 18:16:23 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:16:24.387+0000] {spark_submit.py:634} INFO - 24/12/06 18:16:24 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:16:25.388+0000] {spark_submit.py:634} INFO - 24/12/06 18:16:25 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:16:26.391+0000] {spark_submit.py:634} INFO - 24/12/06 18:16:26 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:16:27.392+0000] {spark_submit.py:634} INFO - 24/12/06 18:16:27 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:16:28.393+0000] {spark_submit.py:634} INFO - 24/12/06 18:16:28 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:16:29.394+0000] {spark_submit.py:634} INFO - 24/12/06 18:16:29 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:16:30.393+0000] {spark_submit.py:634} INFO - 24/12/06 18:16:30 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:16:31.394+0000] {spark_submit.py:634} INFO - 24/12/06 18:16:31 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:16:31.395+0000] {spark_submit.py:634} INFO - 24/12/06 18:16:31 INFO RetryInvocationHandler: java.net.ConnectException: Your endpoint configuration is wrong; For more details see:  http://wiki.apache.org/hadoop/UnsetHostnameOrPort, while invoking ApplicationClientProtocolPBClientImpl.getNewApplication over null after 1 failover attempts. Trying to failover after sleeping for 21110ms.
[2024-12-06T18:16:53.506+0000] {spark_submit.py:634} INFO - 24/12/06 18:16:53 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:16:54.507+0000] {spark_submit.py:634} INFO - 24/12/06 18:16:54 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:16:55.508+0000] {spark_submit.py:634} INFO - 24/12/06 18:16:55 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:16:56.509+0000] {spark_submit.py:634} INFO - 24/12/06 18:16:56 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:16:57.510+0000] {spark_submit.py:634} INFO - 24/12/06 18:16:57 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:16:58.511+0000] {spark_submit.py:634} INFO - 24/12/06 18:16:58 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:16:59.511+0000] {spark_submit.py:634} INFO - 24/12/06 18:16:59 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:17:00.510+0000] {spark_submit.py:634} INFO - 24/12/06 18:17:00 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:17:01.511+0000] {spark_submit.py:634} INFO - 24/12/06 18:17:01 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:17:02.512+0000] {spark_submit.py:634} INFO - 24/12/06 18:17:02 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:17:02.513+0000] {spark_submit.py:634} INFO - 24/12/06 18:17:02 INFO RetryInvocationHandler: java.net.ConnectException: Your endpoint configuration is wrong; For more details see:  http://wiki.apache.org/hadoop/UnsetHostnameOrPort, while invoking ApplicationClientProtocolPBClientImpl.getNewApplication over null after 2 failover attempts. Trying to failover after sleeping for 18904ms.
[2024-12-06T18:17:22.419+0000] {spark_submit.py:634} INFO - 24/12/06 18:17:22 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:17:23.420+0000] {spark_submit.py:634} INFO - 24/12/06 18:17:23 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:17:24.422+0000] {spark_submit.py:634} INFO - 24/12/06 18:17:24 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:17:25.423+0000] {spark_submit.py:634} INFO - 24/12/06 18:17:25 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:17:26.424+0000] {spark_submit.py:634} INFO - 24/12/06 18:17:26 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:17:27.425+0000] {spark_submit.py:634} INFO - 24/12/06 18:17:27 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:17:28.426+0000] {spark_submit.py:634} INFO - 24/12/06 18:17:28 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:17:29.428+0000] {spark_submit.py:634} INFO - 24/12/06 18:17:29 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:17:30.428+0000] {spark_submit.py:634} INFO - 24/12/06 18:17:30 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:17:31.430+0000] {spark_submit.py:634} INFO - 24/12/06 18:17:31 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:17:31.432+0000] {spark_submit.py:634} INFO - 24/12/06 18:17:31 INFO RetryInvocationHandler: java.net.ConnectException: Your endpoint configuration is wrong; For more details see:  http://wiki.apache.org/hadoop/UnsetHostnameOrPort, while invoking ApplicationClientProtocolPBClientImpl.getNewApplication over null after 3 failover attempts. Trying to failover after sleeping for 37082ms.
[2024-12-06T18:18:09.513+0000] {spark_submit.py:634} INFO - 24/12/06 18:18:09 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:18:10.514+0000] {spark_submit.py:634} INFO - 24/12/06 18:18:10 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:18:11.515+0000] {spark_submit.py:634} INFO - 24/12/06 18:18:11 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:18:12.516+0000] {spark_submit.py:634} INFO - 24/12/06 18:18:12 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:18:13.517+0000] {spark_submit.py:634} INFO - 24/12/06 18:18:13 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:18:14.518+0000] {spark_submit.py:634} INFO - 24/12/06 18:18:14 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:18:15.519+0000] {spark_submit.py:634} INFO - 24/12/06 18:18:15 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:18:16.521+0000] {spark_submit.py:634} INFO - 24/12/06 18:18:16 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:18:17.522+0000] {spark_submit.py:634} INFO - 24/12/06 18:18:17 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:18:18.523+0000] {spark_submit.py:634} INFO - 24/12/06 18:18:18 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:18:18.527+0000] {spark_submit.py:634} INFO - 24/12/06 18:18:18 INFO RetryInvocationHandler: java.net.ConnectException: Your endpoint configuration is wrong; For more details see:  http://wiki.apache.org/hadoop/UnsetHostnameOrPort, while invoking ApplicationClientProtocolPBClientImpl.getNewApplication over null after 4 failover attempts. Trying to failover after sleeping for 27776ms.
[2024-12-06T18:18:47.303+0000] {spark_submit.py:634} INFO - 24/12/06 18:18:47 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:18:48.304+0000] {spark_submit.py:634} INFO - 24/12/06 18:18:48 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:18:49.306+0000] {spark_submit.py:634} INFO - 24/12/06 18:18:49 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:18:50.308+0000] {spark_submit.py:634} INFO - 24/12/06 18:18:50 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:18:51.309+0000] {spark_submit.py:634} INFO - 24/12/06 18:18:51 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:18:52.310+0000] {spark_submit.py:634} INFO - 24/12/06 18:18:52 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:18:53.311+0000] {spark_submit.py:634} INFO - 24/12/06 18:18:53 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:18:54.313+0000] {spark_submit.py:634} INFO - 24/12/06 18:18:54 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:18:55.314+0000] {spark_submit.py:634} INFO - 24/12/06 18:18:55 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:18:56.315+0000] {spark_submit.py:634} INFO - 24/12/06 18:18:56 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:18:56.317+0000] {spark_submit.py:634} INFO - 24/12/06 18:18:56 INFO RetryInvocationHandler: java.net.ConnectException: Your endpoint configuration is wrong; For more details see:  http://wiki.apache.org/hadoop/UnsetHostnameOrPort, while invoking ApplicationClientProtocolPBClientImpl.getNewApplication over null after 5 failover attempts. Trying to failover after sleeping for 36633ms.
[2024-12-06T18:19:33.949+0000] {spark_submit.py:634} INFO - 24/12/06 18:19:33 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
